MediaTracker/app/src/main/java/com/izor066/android/mediatracker/ui/fragment/SortItemsDialogFragment.java

```diff
@@ -0,0 +1,115 @@
+package com.izor066.android.mediatracker.ui.fragment;
+
+
+import android.app.Activity;
+import android.app.Dialog;
+import android.os.Bundle;
+import android.support.v4.app.DialogFragment;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.RadioButton;
+import android.widget.RadioGroup;
+
+import com.izor066.android.mediatracker.R;
+
+/**
+ * A simple {@link Fragment} subclass.
+ */
+public class SortItemsDialogFragment extends DialogFragment implements RadioGroup.OnCheckedChangeListener {
+
+    private RadioButton title;
+    private RadioButton author;
+    private RadioButton added;
+    private RadioGroup sortItems;
+    private OnSortingOptionSelectedListener onSortingOptionSelectedListener;
+    private String currentSortCriteria = "added";
+    private boolean isAscending = true;
+
+
+    public SortItemsDialogFragment() {
+        // Required empty public constructor
+    }
+
+    public static interface OnSortingOptionSelectedListener {
```

Comment:
:+1: Nice! This is the way to communicate from a fragment. 

As a nit pick, the interface doesn't need to be `static`.